### DISCUSSION TITLE
No response with mirror announcement

### DISCUSSION THREAD

**Discussion by cicku** [linuxmint/discussions#270] [Development] [answered]
Hello team,

I have emailed the address for a few times and never got any response for my mirror announcement:

- https://linuxmint.com/mirrors.php

Is root@ still working? Can you let me know if you still accept mirrors or not? If yes, when will my request be processed?

**Comment by cicku** [comment]
@clefebvre

**Reply by clefebvre** [reply, marked as answer]
Hi Christopher,

We got your email for the cicku.me mirror. We'll update the list prior to the BETA, it might take a few days.
